### PR TITLE
chore(ci): run visual tests on linux

### PR DESCRIPTION
**Pull request by YonatanKra**

### CODE REVIEW

File: .github/workflows/pull-request.yml [hunk 1]
```diff
@@ -82,3 +82,17 @@ jobs:
             flag-name: Tested on ${{ matrix.os }} / ${{ matrix.browser_name }}
         - run: exit 0
           if: ${{ steps.test.outcome == 'success' }}
+  visual-regression:
+    needs: build
+    runs-on: ubuntu-latest
+    container: drizzt99/vonage:1.0.0
```
Comment (yinonov) [inline .github/workflows/pull-request.yml:88]: can u share details on how this works? what's/who's `drizzt99`?

File: .github/workflows/pull-request.yml [hunk 2]
```diff
@@ -82,3 +82,17 @@ jobs:
...
+    env:
+      GITHUB_TOKEN: ${{ github.token }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '16'
+          cache: 'npm'
+      - run: npm install
+      - run: npm run build && ./scripts/visual-tests/run.tests.sh
```
Comment (yinonov) [inline .github/workflows/pull-request.yml:98]: then why it `needs: build`? to be synchronous?

File: package.json [hunk 3]
```diff
@@ -9,15 +9,18 @@
...
     "test:dev": "karma start config/karma/karma.conf.dev.ts --browsers=Chrome",
     "test:chrome": "karma start config/karma/karma.conf.prod.ts --coverage --browsers=ChromeHeadless",
     "test:firefox": "karma start config/karma/karma.conf.prod.ts --coverage --browsers=FirefoxHeadless",
     "test:safari": "karma start config/karma/karma.conf.prod.ts --coverage --browsers=SafariNative",
     "clean:git": "git clean -dfx",
     "clean": "npm run clean:git",
-    "ui-tests": "http-server -s & playwright test",
+    "ui-tests": "./scripts/visual-tests/run.tests.in.container.sh",
+    "ui-tests:update": "npm run ui-tests -- update",
+    "ui-tests:build": "./scripts/visual-tests/build.image.sh",
```
Comment (yinonov) [inline package.json:22]: when is this used?

File: package.json [hunk 4]
```diff
@@ -9,15 +9,18 @@
...
     "dev": "eleventy --serve",
     "test:dev": "karma start config/karma/karma.conf.dev.ts --browsers=Chrome",
     "test:chrome": "karma start config/karma/karma.conf.prod.ts --coverage --browsers=ChromeHeadless",
     "test:firefox": "karma start config/karma/karma.conf.prod.ts --coverage --browsers=FirefoxHeadless",
     "test:safari": "karma start config/karma/karma.conf.prod.ts --coverage --browsers=SafariNative",
     "clean:git": "git clean -dfx",
     "clean": "npm run clean:git",
-    "ui-tests": "http-server -s & playwright test",
+    "ui-tests": "./scripts/visual-tests/run.tests.in.container.sh",
+    "ui-tests:update": "npm run ui-tests -- update",
```
Comment (yinonov) [inline package.json:21]: no need for `--`?
```suggestion
    "ui-tests:update": "npm run ui-tests -- --update",
```

File: src/components/badge/ui.test.ts [hunk 5]
```diff
@@ -1,9 +1,13 @@
-import { test, expect } from '@playwright/test';
+import { test, expect, Page } from '@playwright/test';
```
Comment (yinonov) [inline src/components/badge/ui.test.ts:1]: said this approach will deprecate once we harvest the READMEs right?

File: package.json (hunk 3 above)
Comment (YonatanKra) [reply package.json:22]: Will remove.

File: package.json [hunk 6]
```diff
@@ -9,15 +9,18 @@
...
+    "ui-tests:build": "./scripts/visual-tests/build.image.sh",
+    "ui-tests:local": "http-server -s & playwright test",
```
Comment (YonatanKra) [inline package.json:23]: Emphasise this can be flaky.

**Review by yinonov — APPROVED** [review]

**Pull request merged**